web/packages/teleterm/src/ui/Search/pickers/ActionPicker.tsx

```diff
@@ -439,7 +480,61 @@ export function TypeToSearchItem() {
   );
 }
 
+export function ResourceSearchErrorsItem(props: {
+  errors: ResourceSearchError[];
+  getClusterName: (resourceUri: uri.ClusterOrResourceUri) => string;
+  onShowDetails: () => void;
+}) {
+  const { errors, getClusterName } = props;
+
+  let shortDescription: string;
+
+  if (errors.length === 1) {
+    const firstErrorMessage = errors[0].messageWithClusterName(getClusterName);
+    shortDescription = `${firstErrorMessage}.`;
+  } else {
+    const allErrorMessages = errors
+      .map(err => firstLowerCase(err.messageWithClusterName(getClusterName)))
```

Comment:
Lowercasing the error message like this is a little bit ehhh as well, I guess I could do something like `err.messageWithClusterName(getClusterName, {capitalize: false})`?

Reply:
I like the current version more, IMO capitalizing the message shouldn't be `ResourceSearchError` responsibility.

Reply:
The problem with manipulating the case outside of `ResourceSearchError` is that if `ResourceSearchError` ever gets updated to include, let's say, the cluster name as the first word in the message, then the rest of the app will keep lowercasing it without even knowing about the change.

I don't actually know if the cluster name can ever have uppercase characters, but I mean the general problem of this nature is quite common. That's why in general we might be better off manipulating the case right at the source.

Reply:
I see, it makes sense to move it to `ResourceSearchError`.